python/ray/experimental/data/impl/arrow_block.py

```diff
@@ -89,6 +89,9 @@ def add(self, item: Union[dict, ArrowRow]) -> None:
         for key, value in item.items():
             self._columns[key].append(value)
 
+    def add_pandas_df(self, df: "pandas.DataFrame") -> None:
```

Comment:
Remove this.

Reply:
Where do you prefer to have this? I prefer not to use pyarrow.Table.from_pandas inside `dataset.py`, but if you think that's the way to go I can do that.

Reply:
Yeah just make the call explicitly instead of adding a method that is just a thin wrapper